src/components/dashboard/Dashboard.tsx

```diff
@@ -1,27 +1,181 @@
+          md: "linear(to-r, #33c2ff, #7b69ec)",
+        }}
+        boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px"
+        alignItems="center"
+        justifyContent="space-around"
+      >
+        <Box
+          color="white"
+          paddingY="32px"
+          paddingLeft={{ base: "16px", md: "32px" }}
+          paddingRight={{ base: "16px", md: "0px" }}
+        >
+          <Heading size="xl" marginBottom="15px">
+            Welcome Back {profile.name?.first}!
+          </Heading>
+          <Text>
+            We’re happy to see you here! We’re currently running our {currentHexathon.name} and we’d
+            love to see you there!
+          </Text>
+          <HStack
+            maxWidth={{ base: "500px", md: "400px" }}
+            justifyContent="space-around"
+            marginTop={{ base: 4, md: 8 }}
+            spacing={4}
+          >
+            <Button width="45%" color="#7B69EC" borderWidth="1px" borderColor="#7B69EC">
+              Apply
+            </Button>
+            <Button width="45%" bgColor="#7B69EC">
+              Details
+            </Button>
+          </HStack>
+        </Box>
+        {application.confirmed ? (
+        <Box
+          border="8px"
+          borderStyle="solid"
+          borderColor="white"
+          borderRadius="3xl"
+          padding="10px"
+          bgColor="#b4c0fa"
+          marginY={{ md: "20px" }}
+          marginBottom={{ base: "40px" }}
+          marginX={{ base: "auto", md: "64px" }}
+        >
+            <QRCodeSVG
+              value={JSON.stringify({
+                uid: user?.uid,
+                name: {
+                  first: profile.name?.first,
+                  last: profile.name?.last,
+                },
+                email: user?.email,
+              })}
+              bgColor="#b4c0fa"
+              size={140}
+            />
+        </Box>
+        ) : (
+          null
+        )
+        }
+      </Flex>
+      <Stack margin={{ base: "20px", md: 0 }} marginBottom={{ base: 0, md: "15px" }}>
+        <Box margin="35px 25px 15px 25px">
+          <Heading fontSize="36px" fontWeight="semibold" marginBottom="10px">
+            Tracks
+          </Heading>
+          <Text>Select one of the tracks from below to apply to {currentHexathon.name}.</Text>
+        </Box>
+        <SimpleGrid columns={branches.length === 0 ? 1 : { base: 1, md: 2 }} spacing={4}>
+          {branches.length === 0 ? (
+            <Text textAlign="center" fontStyle="italic" paddingX="40px">
+              We are currently working hard to finalize the tracks for {currentHexathon.name}!
+              Please check back later!
+            </Text>
+          ) : (
+            branches.map((branch: any) => (
+              <Tile
+                key={branch.name}
+                title={branch.name}
+                description={getDescription(branch)}
+              />
+            ))
+          )}
+        </SimpleGrid>
```

Comment:
One thing I'd recommend doing is maybe put this in it's own component. That way, you can also separate the api request into this component and help keep the component clean. As this component could be pretty simple and just render the list of tiles for branches.